### ISSUE TITLE
Missing Cyber Observable 2 Icons

### ISSUE THREAD

**Issue by sudomedo** [freetaxii/stix2-graphics#4]
Hi,

Would you please create icons "300dpi" for the following Objects:

- Artifact
- ​​Directory
- ​​File
- ​Mutex
- ​Process
- ​Software
- ​User Account
- ​Windows™ Registry Key
- ​X.509 Certificate

**Comment by jordan2175** [comment]
Are you looking for PGNs or SVGs?

**Comment by clenk** [comment]
PNGs of these would be nice.

**Comment by JG-CTIN** [comment]
Any progress on making PNGs for the SCOs listed above?

**Comment by NerdGGuy** [comment]
Yes please

**Comment by macbryc** [comment]
I have finished icons for the remaining SCOs if you want to take a look - I had a graphics artist make some for an internal project we are working on. I will gladly push them up in a pull request if that is the right mechanism. We are using the icons for the STIG project (https://github.com/idaholab/STIG). We are getting ready to push an update that includes all the missing icons.

**Comment by jordan2175** [comment]
A pull request would be great so long as they have the same license as the rest of the icons

**Comment by nicocti** [comment]
It would make sense to have them in a more open licence than CC BY-SA.
STIX being an open specification seeking wide adoption...

**Comment by jordan2175** [comment]
Let me rephrase, as I was not clear. They should be at least CC BY-SA or a license that is more open than that.

**Comment by KnightChaser** [comment]
Is this repository completely abandoned? It looks like those missing icons aren't still updated and the update of this repository paused for a long time...

**Comment by jordan2175** [comment]
I have just been busy. if someone wants to give examples of what the icons should and could look like, I am happy to try and work on making them look like the rest.
